Source/Forms/AppsForm.vb

```diff
@@ -1093,8 +1093,8 @@ Public Class AppsForm
         'crash report by user
         Try
             Dim dir = CurrentPackage.Directory.TrimTrailingSeparator
-            Dim path = Environment.GetEnvironmentVariable("path", EnvironmentVariableTarget.User)
-            path = path.Replace(";;", ";").TrimEnd(";"c)
+            Dim pathVar = If(Environment.GetEnvironmentVariable("path", EnvironmentVariableTarget.User), "")
+            Dim pathItems = pathVar.Split(New String() {Path.PathSeparator}, StringSplitOptions.RemoveEmptyEntries).ToList()
```

Comment:
Can not work, because `Path.PathSeparator` is the separator for folders in the file system whereas in the PATH variable each path is separated with `;;` (on Windows) and that is what we want to achieve.

How do they look like on Linux?

Reply:
> Can not work, because `Path.PathSeparator` is the separator for folders in the file system whereas in the PATH variable each path is separated with `;;` (on Windows) and that is what we want to achieve.
> 
> How do they look like on Linux?

Linux uses the Path Separator like `export PATH="/home/ahmed/toolchains/mingw-w64-x86_64/bin:${PATH}"` to set a new one but i don't think Environment.GetEnvironmentVariable will work there as intended maybe will need so testing to confirm

Reply:
So it's `;` on Windows and `:` on Linux.  🤔 

>  i don't think Environment.GetEnvironmentVariable will work there as intended maybe will need to test it to confirm

That's always a good idea.
I'll also wait a bit to merge this one as there are multiple things to check.

Reply:
> So it's `;` on Windows and `:` on Linux. 🤔
> 
> > i don't think Environment.GetEnvironmentVariable will work there as intended maybe will need to test it to confirm
> 
> That's always a good idea. I'll also wait a bit to merge this one as there are multiple things to check.

i misunderstood you i thought you was taking about how linux treats the path for environment variables with / or \
about the `:` it's part of the command not a separator
linux stores its environment variables in files formated like `key=value` with each one has its own line

Reply:
Okay, so it's a `;` on Windows and a list on Linux as far as I can see.
The problem is I can't tell exactly what type `pathVar` is of on Linux.  😕 

@Valdiralita  Do you know more, since you have ported it at least once on Linux? Or am I wrong?

`Path.PathSeparator` should not work on both systems....

Reply:
[Path.PathSeparator](https://learn.microsoft.com/en-US/dotnet/api/system.io.path.pathseparator?view=netframework-4.8): A platform-specific separator character used to separate path strings in environment variables.
- On windows it returns the character ";"
- On linux native it returns ":"
- On linux with wine it returns ";"

EnvironmentVariableTarget.User is kinda unsupported (https://github.com/dotnet/docs/issues/6277) so ive removed it.

The code gives now the following output:
- On windows it returns paths separated by ";"
- On linux native it returns paths separated by ":"
- On linux with wine it returns paths separated by ";"